dashboard/.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
we should put this in the top-level folder, it'll take effect in the /dashboard subfolder automatically